app/src/main/java/ch/epfl/sdp/peakar/points/Point.java

```diff
@@ -144,6 +145,21 @@ public double getAltitude() {
         return altitude;
     }
 
+    /**
+     * gets the distance relative to the set userPoint
+     * @return distance in meters between point and set userPoint
+     */
+    public double getDistanceToUser(){
```

Comment:
Yes, it's so that we do not have to compute the distance everytime. Like this, we can compute it only once when the POIPoints are computed in the ComputePOIPoints just like the vertical and horizontal bearing.